ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -404,6 +407,642 @@ objects:
+              PrivateHostedZoneName:
+                default: "Private Hosted Zone Name"
+              PrivateHostedZoneId:
+                default: "Private Hosted Zone ID"
+              ClusterName:
+                default: "Cluster name"
+
+        Conditions:
+          DoRegistration: !Equals ["yes", !Ref AutoRegisterELB]
+          DoDns: !Equals ["yes", !Ref AutoRegisterDNS]
+
+        Resources:
+          ProxyIamRole:
+            Type: AWS::IAM::Role
+            Properties:
+              AssumeRolePolicyDocument:
+                Version: "2012-10-17"
+                Statement:
+                - Effect: "Allow"
+                  Principal:
+                    Service:
+                    - "ec2.amazonaws.com"
+                  Action:
+                  - "sts:AssumeRole"
+              Path: "/"
+              Policies:
+              - PolicyName: !Join ["-", [!Ref InfrastructureName, "proxy", "policy"]]
+                PolicyDocument:
+                  Version: "2012-10-17"
+                  Statement:
+                  - Effect: "Allow"
+                    Action: "ec2:Describe*"
+                    Resource: "*"
+
+          ProxyInstanceProfile:
+            Type: "AWS::IAM::InstanceProfile"
+            Properties:
+              Path: "/"
+              Roles:
+              - Ref: "ProxyIamRole"
+
+          ProxySecurityGroup:
+            Type: AWS::EC2::SecurityGroup
+            Properties:
+              GroupDescription: Cluster Proxy Security Group
+              SecurityGroupIngress:
+              - IpProtocol: tcp
+                FromPort: 22
+                ToPort: 22
+                CidrIp: 0.0.0.0/0
+              - IpProtocol: tcp
+                ToPort: 3128
+                FromPort: 3128
+                CidrIp: !Ref AllowedProxyCidr
+              - IpProtocol: tcp
+                ToPort: 3130
+                FromPort: 3130
+                CidrIp: !Ref AllowedProxyCidr
+              - IpProtocol: tcp
+                ToPort: 19531
+                FromPort: 19531
+                CidrIp: !Ref AllowedProxyCidr
+              VpcId: !Ref VpcId
+
+          ProxyInstance:
+            Type: AWS::EC2::Instance
+            Properties:
+              ImageId: !Ref RhcosAmi
+              IamInstanceProfile: !Ref ProxyInstanceProfile
+              InstanceType: "i3.large"
+              NetworkInterfaces:
+              - AssociatePublicIpAddress: "true"
+                DeviceIndex: "0"
+                GroupSet:
+                - !Ref "ProxySecurityGroup"
+                - !Ref "MasterSecurityGroupId"
+                SubnetId: !Ref "PublicSubnet"
+              UserData:
+                Fn::Base64: !Sub
+                - '{"ignition":{"config":{"replace":{"source":"\${S3Loc}","verification":{}}},"timeouts":{},"version":"2.1.0"},"networkd":{},"passwd":{},"storage":{},"systemd":{}}'
+                - {
+                  S3Loc: !Ref ProxyIgnitionLocation
```

Comment:
nit: we are serving this out of our CI cluster, not out of S3.  Can we use a less-opinionated variable name like `ProxyIgntionLocation: !Ref ProxyIgnitionLocation` ;) ?